apps/web/components/verification/verification-workspace.module.css

```
@@ -0,0 +1,14 @@
+.workspace { display: grid; gap: 18px; padding-block: 30px 70px; }
+.hidden { position: absolute; width: 1px; height: 1px; overflow: hidden; clip: rect(0 0 0 0); }
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)